contrib/internal/httptrace/config_test.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package httptrace
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig(t *testing.T) {
+	restore := cleanEnv()
+	defaultCfg := newConfig()
+	restore()
+	for _, tc := range []struct {
+		name      string
+		env       map[string]string
+		cfgSetter func(*config)
```

Comment:
It is not immediately clear how the tests work, so maybe:
```suggestion
		cfgSetter func(*config) // sets *config to the test's expected output
```
Also, why not `cfg` or `out`?

Reply:
Yeah, actually I'll replace this with the direct expected `config` struct.